utils/appliance/__init__.py

```diff
@@ -2198,6 +2207,56 @@ def configure_appliance_for_openldap_ext_auth(self, appliance_fqdn):
             'setenforce 0 && systemctl restart sssd && systemctl restart httpd')
         self.wait_for_web_ui()
 
+    @logger_wrap("Configuring VM Console: {}")
+    def configure_vm_console_cert(self, log_callback=None):
+        """This method generates a self signed SSL cert and installs it
+           in the miq/vmdb/certs dir.   This cert will be used by the
+           HTML 5 VM Console feature.  Note evmserverd needs to be restarted
+           after running this.
+        """
+        log_callback('Installing SSL certificate')
+
+        cert = conf.cfme_data['vm_console'].get('cert', None)
```

Comment:
**Required**
The code quality thing will complain about this I believe. None is the default value for `dict.get()` so its redundant